src/pages/PetCare/ArticleDetail/ArticleDetail.tsx

```
@@ -141,40 +137,40 @@ export default function ArticleDetail() {
+                  className="px-4 py-2  text-[#FF99C0] rounded-full text-sm font-medium hover:bg-pink-200 transition-colors cursor-pointer"
                 >
                   #{tag}
                 </span>
               ))}
             </div>
 
+            {/* Article content */}
+            <div
+              className="prose max-w-none mb-8 prose-headings:text-[#0053A3] prose-a:text-[#FF99C0]"
+              dangerouslySetInnerHTML={{ __html: article.content }}
+            />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)